ipf/src/test/java/org/motechproject/ipf/osgi/InitializerServicesIT.java

```diff
@@ -0,0 +1,30 @@
+package org.motechproject.ipf.osgi;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.motechproject.ipf.service.IPFInitializer;
+import org.motechproject.testing.osgi.container.MotechNativeTestContainerFactory;
+import org.ops4j.pax.exam.ExamFactory;
+import org.ops4j.pax.exam.junit.PaxExam;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerSuite;
+
+import javax.inject.Inject;
+
+import static junit.framework.Assert.assertNotNull;
+
+@RunWith(PaxExam.class)
+@ExamReactorStrategy(PerSuite.class)
+@ExamFactory(MotechNativeTestContainerFactory.class)
+public class InitializerServicesIT {
+
+    @Inject
+    private IPFInitializer ipfInitializer;
```

Comment:
IPFInitializer is not exposed as an OSGi service so it cannot be injected, this causes the test to fail.